lib/i18n-ai.rb

```diff
@@ -6,14 +8,13 @@ module I18nAi
   class Error < StandardError; end
 
   class << self
-    attr_writer :configuration
-
-    def configuration
-      @configuration ||= Configuration.new
-    end
+    attr_accessor :configuration
```

Comment:
do you mind putting back the old code here? seems like there's no change besides using the explicit receive `.self`?